java/server/test/org/openqa/selenium/grid/distributor/DistributorTest.java

```diff
@@ -582,7 +582,7 @@ public void shouldPrioritizeHostsWithTheMostSlotsAvailableForASessionType() {
         );
 
         Session firefoxSession = distributor.newSession(createRequest(firefoxPayload)).getSession();
-        LOG.info(String.format("Firefox Session %d assigned to %s", i, chromeSession.getUri()));
+        LOG.finer(String.format("Firefox Session %d assigned to %s", i, chromeSession.getUri()));
```

Comment:
Since this is in a test, I imagine that the choice of `info` level was deliberate.